spec/easyrsa/03_ca_spec.rb

```diff
@@ -31,14 +31,14 @@
   it 'throw error when bit length is too weak' do
 
     expect {
-      EasyRSA::CA.new("CN=ca/DC=example", 512)
+      EasyRSA::CA.new('CN=ca/DC=example', 512)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.